docs/source/getting-started/updating.adoc

```diff
@@ -24,3 +24,5 @@ This command checks whether there is a newer version of Minishift available.
 If so, it prompts user to confirm the update, downloads the new binary and replaces the current version of Minishift.
 
 NOTE: It is recommended to run the `minishift update` command after the xref:../command-ref/minishift_delete.adoc#[`minishift delete`] to avoid backward compatibility issues.
+
+NOTE: `minishift update` will only work for versions *_1.5.0_* and above (*_>=v1.5.0_*).
```

Comment:
it is not that it won't work before that, but that it quite literally is not available prior 1.5.0. 

NOTE: `minishift update` is available from version *_1.5.0_* and above (*_>=v1.5.0_*).

Reply:
@hferentschik I don't think that's the case. i tried with version 1.3.1 and the `minishift update` command is available.

also @budhrg tried with all the versions till 1.5.0 https://github.com/minishift/minishift/issues/1595#issuecomment-338553741

```
./minishift update
A newer version of minishift is available.
Do you want to update from 1.3.1 to 1.7.0 now? [y/N]: y
Downloading https://github.com/minishift/minishift/releases/download/v1.7.0/minishift-1.7.0-linux-amd64.tgz
 3.80 MiB / 3.80 MiB [====================================================================================================================] 100.00% 0s
 65 B / 65 B [============================================================================================================================] 100.00% 0s
Update failed: mkdir /tmp/download117407506/minishift-extract/minishift-1.7.0-linux-amd64: not a directory
```

Reply:
@hferentschik , update command get introduced from [v1.1.0](https://github.com/minishift/minishift/releases/tag/v1.1.0). So by it is available from v1.1.0.